tools/run_tests/run_microbenchmark.py

```diff
@@ -133,11 +133,9 @@ def collect_perf(bm_name, args):
                            'bins/mutrace/%s' % bm_name,
                            '--benchmark_filter=^%s$' % line,
                            '--benchmark_min_time=20'])
-    with open('bm.perf', 'w') as f:
-      f.write(subprocess.check_output(['sudo', 'perf', 'script', '-i', 'perf.data']))
-    with open('bm.folded', 'w') as f:
-      f.write(subprocess.check_output([
-          '%s/stackcollapse-perf.pl' % flamegraph_dir, 'bm.perf']))
+    subprocess.check_call(['sudo', 'perf', 'script', '-i', 'perf.data', '>', 'bm.perf'], shell=True)
```

Comment:
Probably not causing a problem but does perf here need to be ran under sudo?

I think shouldn't have to after the setup in https://github.com/grpc/grpc/blob/master/tools/gce/linux_performance_worker_init.sh#L157 (unless using kernel probes).